libbeat/.gitignore

```diff
@@ -8,6 +8,7 @@ _output
 /docs/html_docs
 .vagrant
 build
+/dashboards/import_dashboards
```

Comment:
What will this ignore exactly? In case these are the generated dashboards, shouldn't these be in the build directory?